diff -U10 base.orig/data.lua base/data.lua
--- base.orig/data.lua	2017-05-13 21:06:52.419893412 -0400
+++ base/data.lua	2017-05-13 21:13:48.761627830 -0400
@@ -26,6 +26,6 @@
 require("prototypes.tile.noise-layers")
 require("prototypes.autoplace-controls")
-require("prototypes.map-settings")
-require("prototypes.map-gen-presets")
+--require("prototypes.map-settings")
+--require("prototypes.map-gen-presets")
 require("prototypes.tile.tiles")
 require("prototypes.decorative.decoratives")
Common subdirectories: base.orig/locale and base/locale
Common subdirectories: base.orig/migrations and base/migrations
Common subdirectories: base.orig/prototypes and base/prototypes
